lib/cpp/antaris_sdk_environment.cc

```diff
@@ -33,6 +33,7 @@ static char g_CONF_JSON[MAX_FILE_OR_PROP_LEN_NAME]="/opt/antaris/app/config.json
 #define LISTEN_IP_CONF_KEY                  "LISTEN_IP"
 #define PC_API_PORT_CONF_KEY                "PC_API_PORT"
 #define APP_API_PORT_CONF_KEY               "APP_API_PORT"
+#define KEEPALIVE_ENABLE_KEY                "KEEPALIVE"
```

Comment:
can this be replaced by KEEP_ALIVE ?

Reply:
@deepaktawri  It should remain "KEEPALIVE" as that is conventional: https://en.wikipedia.org/wiki/Keepalive

Reply:
yes, even grpc native macros are without _